doc/modules/classes.rst

```diff
@@ -144,16 +145,50 @@ Classes
    covariance.graph_lasso
 
 
-.. _cross_validation_ref:
-
-:mod:`sklearn.cross_validation`: Cross Validation
-=================================================
+:mod:`sklearn.model_selection`: Model Selection
+===============================================
 
-.. automodule:: sklearn.cross_validation
+.. automodule:: sklearn.model_selection
    :no-members:
    :no-inherited-members:
 
-**User guide:** See the :ref:`cross_validation` section for further details.
+**User guide:** See the :ref:`cross_validation`, :ref:`grid_search` and
+:ref:`learning_curve` sections for further details.
+
+Splitters
+---------
+
+.. currentmodule:: sklearn
+
+.. autosummary::
+   :toctree: generated/
+   :template: class.rst
+
+   model_selection.KFold
+   model_selection.LabelKFold
+   model_selection.StratifiedKFold
+   model_selection.LeaveOneLabelOut
+   model_selection.LeavePLabelOut
+   model_selection.LeaveOneOut
+   model_selection.LeavePOut
+   model_selection.ShuffleSplit
+   model_selection.LabelShuffleSplit
+   model_selection.StratifiedShuffleSplit
+   model_selection.PredefinedSplit
+
+Splitter helper
+---------------
+
+.. currentmodule:: sklearn
+
+.. autosummary::
+   :toctree: generated/
+   :template: function.rst
+
+   model_selection.train_test_split
```

Comment:
This one has a function template and that one is a class template... So I renamed the headings to "splitter classes" and "splitter functions" I hope that is fine?